package.json

```diff
@@ -185,6 +185,18 @@
           "expression.ng": "javascript"
         }
       },
+      {
+        "path": "./syntaxes/let-declaration.json",
+        "scopeName": "template.let.ng",
+        "injectTo": [
+          "text.html.derivative",
+          "source.ts"
+        ],
+        "embeddedLanguages": {
+          "text.html": "html",
+          "expression.ng": "javascript"
```

Comment:
I'm pretty sure these can be removed. I have a PR that would clean this up for some of the other scopes (https://github.com/angular/vscode-ng-language-service/pull/1953/commits/066c1d63a445332d34fb96a369d1c069443b338a). Since none of the scopes in the definition have the `text.html` or `expression.ng` names, I don't think this is used.

Reply:
Done.